subsys/bluetooth/controller/ll_sw/ull_llcp_internal.h

```diff
@@ -113,6 +118,15 @@ struct proc_ctx {
 	/* Procedure pause */
 	int pause;
 
+	/* Handle of related connection */
+	uint16_t conn_handle;
```

Comment:
As we discussed Thursday, remove the use of conn_handle and use the conn pointer instead.

Reply:
yeap